daemon/graphdriver/driver_unsupported.go

```diff
@@ -1,7 +1,14 @@
-// +build !linux
+// +build !linux,!windows
```

Comment:
By the end of these PRs, windows will be supported :) Windows currently doesn't use the daemon build tag, so this code won't currently be compiled in. There's a bunch of PRs to go before we get to the stage where the daemon build tag can be added and all this code I'm adding gets compiled in. Hence I'd prefer that we keep it like this to avoid future PRs to revert that. Hope that makes sense.